-------------------------------------------------------------------------------
Test set: TakingOrdersTests
-------------------------------------------------------------------------------
Tests run: 5, Failures: 5, Errors: 0, Skipped: 0, Time elapsed: 3.492 s <<< FAILURE! - in TakingOrdersTests
acceptanceOfNewOrderWithWrongCourierId(TakingOrdersTests)  Time elapsed: 1.008 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
JSON path message doesn't match.
Expected: is "Курьера с таким id не существует"
  Actual: Not Found.

	at TakingOrdersTests.acceptanceOfNewOrderWithWrongCourierId(TakingOrdersTests.java:54)

acceptanceOfNewOrderWithWrongOrderId(TakingOrdersTests)  Time elapsed: 0.373 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
JSON path message doesn't match.
Expected: is "Заказа с таким id не существует"
  Actual: Not Found.

	at TakingOrdersTests.acceptanceOfNewOrderWithWrongOrderId(TakingOrdersTests.java:62)

acceptanceOfNewOrderWithoutCourierId(TakingOrdersTests)  Time elapsed: 0.818 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Expected status code <400> but was <404>.

	at TakingOrdersTests.acceptanceOfNewOrderWithoutCourierId(TakingOrdersTests.java:42)

successfulAcceptanceOfNewOrder(TakingOrdersTests)  Time elapsed: 0.837 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Expected status code <200> but was <404>.

	at TakingOrdersTests.successfulAcceptanceOfNewOrder(TakingOrdersTests.java:30)

acceptanceOfNewOrderWithoutOrderId(TakingOrdersTests)  Time elapsed: 0.446 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Expected status code <400> but was <404>.

	at TakingOrdersTests.acceptanceOfNewOrderWithoutOrderId(TakingOrdersTests.java:70)

